### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**
[DEMO LINK](https://YevhenOstrovskiy.github.io/react_movies-list-add-form/)

### CODE REVIEW

File: src/components/NewMovie/NewMovie.tsx [hunk 1]
```diff
@@ -48,6 +83,19 @@ export const NewMovie = () => {
...
+                {
+                  title,
+                  description,
+                  imgUrl,
+                  imdbUrl,
+                  imdbId,
+                },
+              );
+              clearForm();
+            }}
```
Comment (sasholk) [inline src/components/NewMovie/NewMovie.tsx:98]: better make handler if callback more than 1 row

File: src/components/NewMovie/NewMovie.tsx [hunk 2]
```diff
@@ -1,45 +1,80 @@
...
+    setImdbUrl('');
+    setImdbId('');
+    setCount(() => count + 1);
+  }
+
+  const handleAddForm = (event: React.FormEvent) => {
+    event.preventDefault();
+  };
+
+  const isDisabled = !(title && imgUrl && imdbUrl && imdbId);
```
Comment (sasholk) [inline src/components/NewMovie/NewMovie.tsx:30]: add method `trim` cuz your form works with only spaces

**Review by sasholk — APPROVED** [review]
Nice Job Mate!! 🔥🔥🔥🔥
But make some changes

File: src/components/NewMovie/NewMovie.tsx (hunk 1 above)
Comment (olya-shyian) [inline src/components/NewMovie/NewMovie.tsx:98]: I would recommend put all this logic to `handleAddForm` function

**Review by olya-shyian — CHANGES_REQUESTED** [review]
Good work!💪

Unfortunately, there is a bug. If you add some space to each input then the 'add' button will no longer be disabled so you can create an empty movie and add to the list.

image

Please, try to fix it

**Review by olya-shyian — APPROVED** [review]
Nice🌸

**Pull request closed without merging**